emails/migrations/0042_domainaddress_used_on.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-05-19 14:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emails', '0041_relayaddress_used_on'),
```

Comment:
Nit: run `black` on this file

Reply:
/me shakes fist as double-quotes.

Reply:
https://github.com/mozilla/fx-private-relay/pull/1968/commits/87f4b1c0d6783f96ea5c4986088085c4eae41d79